students/ReemAlqaysi/Lesson02/Grid.py

```diff
@@ -0,0 +1,55 @@
+""" Grid Printer Exercise for Lesson 2"""
+#by Reem Alqaysi
+"""function that draws a grid like the following:
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+"""
+def print_grid_1():
+	#rows loop
+	row = ("+" + ("-"*4))*2
+	for i in range(0,2):
+		print (row + "+")
+		#"|" loop
+		for v in range(0,4):
+			print (("|"+" "*4)*2+"|")
```

Comment:
make sure you stay consistent with your indentation, it should be exactly 4 spaces.